{"uuid": "b089a2fa-8747-4e31-87e6-85578ae564da", "counter": 7, "stdout": "\u001b[0;31m8.130.45.15 | FAILED | rc=1 >>\u001b[0m\r\n\u001b[0;31mtouch: cannot touch \u2018/root/1.txt\u2019: Permission deniednon-zero return code\u001b[0m", "start_line": 4, "end_line": 6, "runner_ident": "2324dce3-2833-42dd-b6a6-fdcf78ed16c3", "event": "runner_on_failed", "ad_hoc_command_id": 1, "pid": 104646, "created": "2024-07-22T19:17:03.599168+00:00", "parent_uuid": "000c29ae-5ed9-4fa2-7e3e-00000000000a", "event_data": {"playbook": "__adhoc_playbook__", "playbook_uuid": "5a481ca0-b7e4-4c0b-b12d-5f5aca3c085d", "task": "command", "task_uuid": "000c29ae-5ed9-4fa2-7e3e-00000000000a", "task_action": "command", "resolved_action": "command", "task_args": "", "task_path": "", "host": "8.130.45.15", "remote_addr": "8.130.45.15", "res": {"msg": "non-zero return code", "cmd": "touch  /root/1.txt", "stdout": "", "stderr": "touch: cannot touch \u2018/root/1.txt\u2019: Permission denied", "rc": 1, "start": "2024-08-05 14:09:03.748679", "end": "2024-08-05 14:09:03.752191", "delta": "0:00:00.003512", "changed": true, "invocation": {"module_args": {"_raw_params": "touch  /root/1.txt", "_uses_shell": true, "warn": true, "stdin_add_newline": true, "strip_empty_ends": true, "argv": null, "chdir": null, "executable": null, "creates": null, "removes": null, "stdin": null}}, "warnings": ["Consider using the file module with state=touch rather than running 'touch'.  If you need to use command because file is insufficient you can add 'warn: false' to this command task or set 'command_warnings=False' in ansible.cfg to get rid of this message."], "stdout_lines": [], "stderr_lines": ["touch: cannot touch \u2018/root/1.txt\u2019: Permission denied"], "ansible_facts": {"discovered_interpreter_python": "/usr/bin/python"}, "_ansible_no_log": false}, "start": "2024-07-22T19:17:01.831995+00:00", "end": "2024-07-22T19:17:03.598966+00:00", "duration": 1.766971, "ignore_errors": null, "event_loop": null, "uuid": "b089a2fa-8747-4e31-87e6-85578ae564da"}}